File: CSHTML5.Samples.Showcase/Other/ChildWindowHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Controls;

namespace CSHTML5.Samples.Showcase
{
    static class ChildWindowHelper
    {
        public static void ShowChildWindow(UIElement content)
        {
            var childWindow = new ChildWindow()
            {
                Content = content,
                Style = (Style)App.Current.Resources["ViewMoreChildWindow_Style"]
            };
            childWindow.Show();
        }
    }
}

File: src/Other/Internals/ChildWindowHelper.vb
Imports System.Windows
Imports System.Windows.Controls

Namespace Global.OpenSilver.Samples.Showcase
    Friend Module ChildWindowHelper
        Public Sub ShowChildWindow(ByVal content As UIElement)
            Dim childWindow = New ChildWindow() With {
    .Content = content,
    .Style = CType(Application.Current.Resources("ViewMoreChildWindow_Style"), Style)
}
            childWindow.Show()
        End Sub
    End Module
End Namespace
